### PR TITLE
Scenario for slideshow

### PR DESCRIPTION
**Pull request by alectykvach**

### CODE REVIEW

File: tests/steps/login.js [hunk 1]
```diff
@@ -18,6 +18,11 @@ module.exports = () => {
       const loggedInPageSelector = 'body.apos-workflow-live-page';
 
       client.expect.element(loggedInPageSelector).to.be.present;
+
+      // do not collapse admin bar automatically
+      client.execute(function() {
+        apos.adminBar.autoCollapse = function() {};
```
Comment (boutell) [inline tests/steps/login.js]: Is this change necessary - is it not possible for the tests to interact with this with the timing that a human would...?

Comment (alectykvach) [reply tests/steps/login.js]: We can remove it. It’s to avoid problems in the future.

**Comment by alectykvach** [comment]
@boutell 🆙
